.github/workflows/containers.yml

```diff
@@ -0,0 +1,38 @@
+name: Containers
+on:
+  # Be able to run the job manually when needed
+  # (Actions -> Containers -> Run workflow)
+  workflow_dispatch:
+  # Build every week on Monday 00:00
+  schedule:
+    - cron:  '0 0 * * 1'
+env:
+  IMAGE_REGISTRY: quay.io
+  can_push: ${{ github.repository_owner == 'oVirt' }}
+jobs:
+  container:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        container: [ centos-8 centos-9 fedora-35 fedora-36 ]
+    steps:
+      - uses: actions/checkout@v2
+      - name: Install dependencies
+        run: |
+          sudo apt update
+          sudo apt install podman
+      - name: Build container images
+        working-directory: automation/containers
+        run: make ${{ matrix.container }}
+      - name: Push to Quay.io
+        id: push-to-quay
+        uses: redhat-actions/push-to-registry@v2
+        with:
+          image: ovirt/${{ matrix.container }}
+          tags: ${{ matrix.container }}
+          registry: ${{ env.IMAGE_REGISTRY }}
+          username: ${{ secrets.QUAY_USERNAME  }}
+          password: ${{ secrets.QUAY_TOKEN }}
+      - name: Print image url
+        if: ${{ env.can_push == 'true' }}
+        run: echo "Image pushed to ${{ steps.push-to-quay.outputs.registry-paths }}"
```

Comment:
So we try to push even when we cannot push?

I expect that we use redhat-actions/push-to-registry@v2 only if we can push.

Reply:
Yes, my bad, missed that line.